lruttl/lru.py

```diff
@@ -26,11 +26,20 @@ def set(self, key, value, ttl=None):
         item = {'value': value,
                 'expires': expires}
         self.cache[key] = item
+        
+    if hasattr(OrderedDict, 'move_to_end'):
+        # python 3.2+
+        def _move_to_end(self, key):
+            self.cache.move_to_end(key)
+    else:
+        # python 2.7
+        def _move_to_end(self, key):
+            self.cache[key] = self.cache.pop(key)
```

Comment:
I think the methods look fine, but I think it a bit strange to have the define with the if statements. What about:

```
def _move_to_end(self, key):
    hasattr(OrderedDict, 'move_to_end'):
        self.cache.move_to_end(key)
    else:
        self.cache[key] = self.cache.pop(key)
```

Reply:
Your version will re-calculate `hasattr()` every time `_move_to_end` is launched, while *def-within-if* approach will calculate it only once. It is a tiny optimization which is probably not important in this case.